mrbelvedereci/plan/handlers.py

```diff
@@ -2,8 +2,31 @@
 from django.conf import settings
 from django.db.models.signals import post_save
 from django.dispatch import receiver
+from django.contrib.admin.models import LogEntry, ADDITION
+from django.contrib.contenttypes.models import ContentType
 from github3 import login
 from mrbelvedereci.plan.models import Plan
+from mrbelvedereci.build.models import Build
+
+
+@receiver(post_save, sender=Build)
+def log_manual_build(sender, **kwargs):
```

Comment:
It was really just to play around with LogEntry. Since the admin log only has three action types - Created, Changed, and Deleted, it doesn't have a very expressive or useful UI. there's also no out of the box Lists of LogEntry in the admin interface. 

I don't think it's a great solution for any of our audit logging needs, really. I'll remove it from the PR.